jmclient/jmclient/configure.py

```diff
@@ -530,8 +532,10 @@ def get_blockchain_interface_instance(_config):
             rpc_wallet_file)
         if source == 'bitcoin-rpc': #pragma: no cover
             bc_interface = BitcoinCoreInterface(rpc, network)
-        else:
+        elif source == 'regtest':
             bc_interface = RegtestBitcoinCoreInterface(rpc)
+        else:
```

Comment:
I think this should be an elif with an error raised in the 'else'.

Reply:
Done